xpdan/data_reduction.py

```diff
@@ -80,7 +80,7 @@ def _feature_gen(self, event):
         for key in self.data_fields:
             val = event['data'].get(key, None)
             if el is not None:
-                feature = "{}={}".format(key, val)
+                feature = "{}".format(val)
```

Comment:
I mean it would be a good test/functionality to have, do you mind putting it into a separate PR?